src/tag_index.c

```diff
@@ -134,8 +136,8 @@ size_t TagIndex_Index(TagIndex *idx, const char **values, size_t n, t_docId docI
 }
 
 typedef struct {
+  TagIndex *idx;
   IndexIterator **its;
-  uint32_t uid;
```

Comment:
We do not use this?

Reply:
@MeirShpilraien 
Since I use `TagIndex` with `TagIndex_OpenIndex` to retrieve the inverted index of the tag value.